src/server/rewrites.js

```diff
@@ -0,0 +1,64 @@
+var fs = require('fs');
+var logger = require('./logging').logger;
+var PathService = require('../services/path');
+
+
+/**
+ * @todo This same logic of reading JSON files is in a few places. It should be
+ *       abstracted into something like `ConfigService.readConfigFile(fileName)`
+ */
+var REWRITES_FILE = 'rewrites.json';
+var SITE_KEY = 'developer.rackspace.com';
+
+try {
+    var rewrites = JSON.parse(fs.readFileSync(
+        PathService.getConfigPath(REWRITES_FILE),
+        'utf-8'
+    ))[SITE_KEY];
```

Comment:
That makes sense. The hardcoding here (and basically all the other screaming snake-case vars) was a major hack until I could have it dynamically choose the site/domain.